### PR TITLE
Prep changelog for 1.0.0-beta.3 release

### PR DESCRIPTION
**Pull request by nmuesch**
### What does this PR do?

See title

### Review checklist
Please check relevant items below:
- [ ] This PR includes all [newly recorded cassettes](/DEVELOPMENT.md) for any modified tests.

- [ ] This PR does not rely on API client schema changes.
    - [ ] The CI should be fully passing.
- [ ] Or, this PR relies on API schema changes and this is a Draft PR that includes tests.
  - Note: CI shouldn't be run on this Draft PR, as its expected to fail without the corresponding schema changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # CHANGELOG
 
+## datadog-api-client-1.0.0-beta.3 / 2020-05-20
+
+* [Added] Add security monitoring to clients. See [#306](https://github.com/DataDog/datadog-api-client-java/pull/306).
+* [Added] Add /v1/validate endpoint. See [#297](https://github.com/DataDog/datadog-api-client-java/pull/297).
+* [Added] Add a .generated_files. See [#274](https://github.com/DataDog/datadog-api-client-java/pull/274).
```
Comment (nmuesch) [inline CHANGELOG.md:7]: This doesn't impact functionality, but its an addition to the repo, so I'm not sure it needs a callout, so I left this.

Comment (bkabrda) [reply CHANGELOG.md:7]: I don't see a problem with keeping this changelog line.

File: RELEASING.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Note that once the release process is started, nobody should be merging/pushing
 
 ### Commands
 
-- See changes ready for release by running `ddev release show changes .` at the root of this project. Add any missing labels to PRs if needed.
+- See changes ready for release by running `ddev release show changes . --tag-prefix datadog-api-client-` at the root of this project. Add any missing labels to PRs if needed.
```
Comment (jirikuncar) [inline RELEASING.md:22]: 👍 🙇‍♂️

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
 # CHANGELOG
 
+## datadog-api-client-1.0.0-beta.3 / 2020-05-20
```
Comment (nmuesch) [inline CHANGELOG.md:3]:
```suggestion
## datadog-api-client-1.0.0-beta.3 / 2020-05-21
```

**Pull request merged**